public/app/features/integrated-alerting/components/Alerts/AlertsActions/AlertsActions.styles.ts

```diff
@@ -0,0 +1,69 @@
+import { css } from 'emotion';
+import { GrafanaTheme } from '@grafana/data';
+
+export const getStyles = (theme: GrafanaTheme) => {
+  const size = 16;
+
+  const hoverColor = theme.isLight ? theme.palette.gray95 : theme.palette.gray15;
+
+  return {
+    button: css`
```

Comment:
Could you please put this todo here:
`TODO: remove the button styles and use an IconButton once implemented as part of the @percona/platform-core library`

Reply:
Done